src/tests/functional/shared_test_classes/src/single_op/non_max_suppression.cpp

```diff
@@ -148,5 +148,251 @@ void Nms9LayerTest::SetUp() {
     function = std::make_shared<ov::Model>(nms, params, "NMS");
 }
 
+
+void NmsLayerTest::compare(const std::vector<ov::Tensor>& expected, const std::vector<ov::Tensor>& actual) {
+    CompareBBoxes(expected, actual);
+}
+
+
+namespace {
+
+typedef struct Rect {
+    int32_t x1;
+    int32_t y1;
+    int32_t x2;
+    int32_t y2;
+} Rect;
+
+class Box {
+public:
+    Box() = default;
+
+    Box(int32_t batchId, int32_t classId, int32_t boxId, Rect rect, float score) {
+        this->batchId = batchId;
+        this->classId = classId;
+        this->boxId = boxId;
+        this->rect = rect;
+        this->score = score;
+    }
+
+    int32_t batchId;
+    int32_t classId;
+    int32_t boxId;
+    Rect rect;
+    float score;
+};
```

Comment:
```suggestion
struct Box {
    int32_t batchId;
    int32_t classId;
    int32_t boxId;
    Rect rect;
    float score;
};
```
Can be replaced by struct?

Reply:
No, there is not difference but code is simpler no need to defines ctors. Is optional.

Reply:
https://godbolt.org/z/vWE9fMWs3 - as example. I cannot use Box(...) without explicitly defined ctor even if it is struct.

Reply:
> https://godbolt.org/z/vWE9fMWs3 - as example. I cannot use Box(...) without explicitly defined ctor even if it is struct.

But looks like initializer_list may be used, right?

Reply:
I converted it to struct.
But anyway I don't see any advantages to it :)